knora/dsplib/schemas/properties-only.json

```diff
@@ -82,6 +82,7 @@
                                 "ListValue",
                                 "Region",
                                 "Resource",
+                                "Representation",
```

Comment:
In the main schema, `Representation` had been added earlier, but it was forgotten here.